yt_dlp/extractor/theta.py

```diff
@@ -0,0 +1,44 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class ThetaIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?theta\.tv/(?P<id>[a-z0-9]+)'
+    _TESTS = [{
+        'url': 'https://www.theta.tv/davirus',
+        'info_dict': {
+            'id': 'DaVirus',
+            'ext': 'mp4',
+            'title': r're:.+',
```

Comment:
Add proper title
If it's big, md5 hash it

Reply:
What if channel owner changes the title?

Reply:
It's not something common and the next time someone works on this, he can just update the tests.